net: sockets: tls: Add error log when cert parse fails

noahp opened the pull request:
Add an error log if there's a failure when attempting to load certificates during socket connect.

Signed-off-by: Noah Pendleton <noah.pendleton@gmail.com>

rlubos reviewed (commented):
The change looks ok, but it seems that you need to update your `user.email` git config and update the commit (with `--reset-author` option) to fix the compliance.

noahp committed:
net: sockets: tls: Add error log when cert parse fails

Add an error log if there's a failure when attempting to load
certificates during socket connect.

Signed-off-by: Noah Pendleton <noah.pendleton@gmail.com>

noahp commented:
@rlubos Should be updated correctly now, sorry about that!

henrikbrixandersen commented:
@noahp Please rebase on current `main`.

noahp commented:
> @noahp Please rebase on current `main`.

@henrikbrixandersen rebased!
